automated_testing/test/tests/auth-tests.ts

```diff
@@ -0,0 +1,65 @@
+import IndexPage from '../pageobjects/index-page';
+import AuthPage from '../pageobjects/auth-page';
+
+describe.skip('Testing AuthPage', () => {
+    const validUserCredentials = {
+        username: 'testuser123',
+        password: '123456',
+    };
+
+    const invalidUserCredentials = {
+        username: 'testuser12',
+        password: '123456',
+    };
+
+    const invalidPasswordCredentials = {
+        username: 'testuser123',
+        password: '12345',
+    };
+
+    const loginWithCredentials = async (credentials) => {
+        const { username, password } = credentials;
+        await AuthPage.performLogIn(username, password);
+        return validUserCredentials;
+    };
+
+    const loginInWithValidCredentials = () => loginWithCredentials(validUserCredentials);
+
+    const loginInWithInvalidCredentials = () => loginWithCredentials(invalidUserCredentials);
+
+    beforeEach(async () => {
+        try {
+            console.log(' --- Trying to logout ---');
+            await AuthPage.performLogOut();
+            console.log(' --- Success logout ---');
+        } catch (err) {
+            console.log(' --- Failed logout ---');
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            const x = 5;
+        }
+    });
+
+    it('Expect successful LogIn', async () => {
+        await AuthPage.open();
+        await loginInWithValidCredentials();
+
+        const myTestsBtn = await AuthPage.authLoginButton;
+        await expect(myTestsBtn).toExist();
+    });
+
+    xit('Expect logOut button to exist', async () => {
```

Comment:
All tests should be active, i.e. not `xit` but `it` when in PR.
Disabling tests is for local environments only